store/etcd/etcd.go

```diff
@@ -450,6 +453,7 @@ func (s *Etcd) WatchTreeExt(directory string, stopCh <-chan struct{}) (<-chan *s
 				LastIndex: result.Node.ModifiedIndex,
 				Action:    result.Action,
 				Dir:       result.Node.Dir,
+				Response:  result,
```

Comment:
It was done to maintain compatibility with `KVPairExt` consumers and to imitate your original commit to a degree.

However, since you asked what do you think would be a better thing to do - export every field of response or export response itself?